data/json/npcs/exodii/exodii_merchant_talk.json

```
@@ -1297,5 +1316,80 @@
       { "text": "I've got some stuff I'd like to install.", "topic": "TALK_DONE", "effect": "bionic_install" },
       { "text": "Would you be able to take out one of my implants?", "topic": "TALK_DONE", "effect": "bionic_remove" }
     ]
+  },
+  {
+    "id": "TALK_EXODII_MERCHANT_You_Buy_Riot_Gear",
+    "type": "talk_topic",
+    "dynamic_line": "[yeah i think we got some that'll be $400]",
+    "responses": [
+      {
+        "text": "[deal]",
+        "effect": [
+          {
+            "u_spend_cash": 40000,
+            "true_eocs": {
+              "id": "exodii_buy_riot_gear_success",
+              "effect": [
+                { "u_spawn_item": "armor_riot", "count": 5 },
+                { "u_spawn_item": "helmet_riot", "count": 5 },
+                { "u_spawn_item": "armor_riot_arm", "count": 5 },
+                { "u_spawn_item": "armor_riot_leg", "count": 5 },
+                { "npc_add_var": "npc_bought_riot_gear", "type": "dialogue", "context": "exodii", "value": "yes" }
+              ]
+            }
+          }
+        ],
+        "topic": "TALK_EXODII_MERCHANT_Talk"
+      },
+      {
+        "text": "[ayo I aint got that cash can I buy on credit?]",
+        "topic": "TALK_EXODII_MERCHANT_You_Buy_Riot_Gear_On_Credit"
+      },
+      { "text": "Can we talk about something else?", "topic": "TALK_EXODII_MERCHANT_Talk" },
+      { "text": "I've gotta go.  See you later, Rubik.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_EXODII_MERCHANT_You_Buy_Riot_Gear_On_Credit",
+    "type": "talk_topic",
+    "dynamic_line": "[okie dokie you got 20 days to pay me back.]",
```

```suggestion
    "dynamic_line": "If'n you survive, We can give you a fortnight and six odd days to pay us'n back.",
```
Taking some shots at some of Rubik's dialogue.